src/Components/Display.jsx

```diff
@@ -1,16 +1,30 @@
-import React, {useState} from 'react';
+import React, { useState, useEffect } from 'react';
 
 function Display(props) {
-    // const [display,setDisplay] = useState(0);
+  const [displayColor, setDisplayColor] = useState('#000000');
 
-    return (
-        <section className="display">
-            This is the Display for the Counter.
-        </section>
-    )
+  function toggleColor() {
+    displayColor === '#000000'
+      ? setDisplayColor('#61dafb')
+      : setDisplayColor('#000000');
+  }
 
+  useEffect(() => {
+    return () => toggleColor();
```

Comment:
so you want this to toggle BEFORE the component mounts?  Impressed that you picked up this trick from our morning sessions.  nice work there.  Have you tried simplifying the function to make it a one liner without the return keyword?

Reply:
```
useEffect( () => () => toggleColor() );
```